purchase_landed_cost/wizard/picking_import_wizard.py

```diff
@@ -50,8 +49,8 @@ def action_import_picking(self):
             self.env.context["active_id"]
         )
         previous_moves = distribution.mapped("cost_lines.move_id")
-        for move in self.mapped("pickings.move_lines"):
-            if move not in previous_moves:
+        for move in self.mapped("pickings.move_line_ids"):
```

Comment:
```suggestion
        for move in self.mapped("pickings.move_ids"):
```

and subsequent changes

Reply:
fix it, thanks!